src/commands/buildCommands/browser.js

```diff
@@ -11,34 +13,49 @@ const {
 
 const getBundlePath = getRelativeBundlePath('app', paths.output.browser);
 
-const buildBrowserApp = localeCode => () => {
+const buildBrowserApp = (localeCode, babel) => () => {
```

Comment:
let's document the new arg so that people know what it looks like. I was wondering if we might want to use `joi` to validate the passed-in config, but that's probably overkill for now